sections/main-product.liquid

```diff
@@ -471,7 +471,7 @@
                   data-quantity = "2",
                   class="product-form__submit button custom-button-secondary_color"
                 >
-                  Add to cart 2
+                  Add to cart 2 - Changes from feature branch
```

Comment:
Hey @mmdmrishabh  this is not the correct way to do this thing please change this to `let`

Reply:
Hey dude it was not an issue just fixed